package.json

```diff
@@ -21,21 +21,23 @@
   },
   "dependencies": {
     "@kth/reqvars": "^2.0.1",
-    "bunyan": "^1.8.12",
+    "bluebird": "^3.7.2",
```

Comment:
Wait a minute, we only use bluebird in one place, and that is in server/utils.js. It's only used to get the promisify function, which nowadays also is included in node by default. Better to use that one and get rid of this dependency.